builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -1195,6 +1202,10 @@ func websiteEndpoint(s3conn *s3.S3, d *schema.ResourceData) (*S3Website, error)
 	return WebsiteEndpoint(bucket, region), nil
 }
 
+func bucketDomainName(bucket string) string {
+	return fmt.Sprintf("%s.s3.amazonaws.com", bucket)
```

Comment:
Hi @stack72, 
Well, I think so. While looking at [this documentation](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/DownloadDistS3AndCustomOrigins.html), it is stated under `The bucket is not configured as a website endpoint` that:
> In general, use the following format:
>
>    bucket-name.s3.amazonaws.com
>
> If your bucket is in the US Standard region and you want Amazon S3 to route requests to a facility in Northern Virginia, use the following format:
>
>    bucket-name.s3-external-1.amazonaws.com
>
> When you specify the bucket name in this format, you can use the following CloudFront features:
> 
> Configure CloudFront to communicate with your Amazon S3 bucket using SSL. For more information, see Using HTTPS with CloudFront.
> Use an origin access identity to require that your users access your content using CloudFront URLs, not by using Amazon S3 URLs. For more information, see Using an Origin Access Identity to Restrict Access to Your Amazon S3 Content.
> Update the content of your bucket by submitting POST and PUT requests to CloudFront. For more information, see HTTP Methods in the topic How CloudFront Processes and Forwards Requests to Your Amazon S3 Origin Server.

Tried creating a S3 bucket in "US Standard" and a CloudFront distrib from the console, and it tries to get the bucket using: `myname.s3.amazonaws.com`.